cita-chain/src/main.rs

```diff
@@ -133,8 +134,8 @@ fn main() {
 
     let nosql_path = DataPath::nosql_path();
     trace!("nosql_path is {:?}", nosql_path);
-    let db_config = DatabaseConfig::with_columns(db::NUM_COLUMNS);
-    let db = Database::open(&db_config, &nosql_path).unwrap();
+    let db_config = DatabaseConfig::with_category_num(db::NUM_COLUMNS);
+    let db = RocksDB::open(&nosql_path, &db_config).unwrap();
```

Comment:
`expect` is better than `unwrap`, we need to know what happen when the system crash.

Reply:
It's anothere problem.

I just replace the old logic with cita-databse

Reply:
ok

Reply:
#704